lib/suse/toolkit/utilities.rb

```diff
@@ -21,7 +21,8 @@ def system_auth
 
       rescue
         raise SUSE::Connect::CannotBuildBasicAuth,
-              "Cannot read username and password from #{SUSE::Connect::Credentials::GLOBAL_CREDENTIALS_FILE}"
+              "\nCannot read username and password from #{SUSE::Connect::Credentials::GLOBAL_CREDENTIALS_FILE}. " \
+              "Has this system been registered already?"
```

Comment:
Haven't tested your code or anything... just about this error message:

- "Has this system been registered already?" for me sounds as you are implying I should register it first to make this action possible... well, in my use case of `--de-register` this might technically true, but still, makes no sense.

Also this might not be the right place to print such a specific error message. Not sure where else `CannotBuildBasicAuth` gets raised. So probably you should check if the system is registered and exit the command on a higher level.